src/contracts/misc/GhoStewardV2.sol

```diff
@@ -0,0 +1,268 @@
+        newBucketCapacity,
+        currentBucketCapacity
+      ),
+      'INVALID_BUCKET_CAPACITY_UPDATE'
+    );
+
+    _facilitatorsBucketCapacityTimelocks[facilitator] = uint40(block.timestamp);
+
+    IGhoToken(GHO_TOKEN).setFacilitatorBucketCapacity(facilitator, newBucketCapacity);
+  }
+
+  /// @inheritdoc IGhoStewardV2
+  function updateGhoBorrowRate(
+    uint256 newBorrowRate
+  ) external onlyRiskCouncil notLocked(_ghoBorrowRateLastUpdated) {
+    DataTypes.ReserveData memory ghoReserveData = IPool(
+      IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPool()
+    ).getReserveData(GHO_TOKEN);
+    require(
+      ghoReserveData.interestRateStrategyAddress != address(0),
+      'GHO_INTEREST_RATE_STRATEGY_NOT_FOUND'
+    );
+
+    uint256 currentBorrowRate = GhoInterestRateStrategy(ghoReserveData.interestRateStrategyAddress)
+      .getBaseVariableBorrowRate();
+    require(
+      _isChangePositiveAndIncreaseLowerThanMax(
+        currentBorrowRate,
+        newBorrowRate,
+        GHO_BORROW_RATE_CHANGE_MAX
+      ) && newBorrowRate <= GHO_BORROW_RATE_MAX,
+      'INVALID_BORROW_RATE_UPDATE'
+    );
+    address cachedStrategyAddress = _ghoBorrowRateStrategiesByRate[newBorrowRate];
+
+    if (cachedStrategyAddress == address(0)) {
+      GhoInterestRateStrategy newRateStrategy = new GhoInterestRateStrategy(
+        POOL_ADDRESSES_PROVIDER,
+        newBorrowRate
+      );
+      cachedStrategyAddress = address(newRateStrategy);
+
+      _ghoBorrowRateStrategiesByRate[newBorrowRate] = cachedStrategyAddress;
+      _ghoBorrowRateStrategies.add(cachedStrategyAddress);
+    }
+
+    _ghoBorrowRateLastUpdated = uint40(block.timestamp);
+
+    IPoolConfigurator(IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPoolConfigurator())
+      .setReserveInterestRateStrategyAddress(GHO_TOKEN, cachedStrategyAddress);
+  }
+
+  /// @inheritdoc IGhoStewardV2
+  function updateGsmExposureCap(
+    address gsm,
+    uint128 newExposureCap
+  ) external onlyRiskCouncil notLocked(_gsmTimelocksByAddress[gsm].gsmExposureCapLastUpdated) {
+    uint128 currentExposureCap = IGsm(gsm).getExposureCap();
+    require(
+      _isChangePositiveAndIncreaseLowerThanMax(
+        currentExposureCap,
+        newExposureCap,
+        currentExposureCap
+      ),
+      'INVALID_EXPOSURE_CAP_UPDATE'
+    );
+    _gsmTimelocksByAddress[gsm].gsmExposureCapLastUpdated = uint40(block.timestamp);
+    IGsm(gsm).updateExposureCap(newExposureCap);
+  }
+
+  /// @inheritdoc IGhoStewardV2
+  function updateGsmFeeStrategy(
+    address gsm,
+    uint256 buyFee,
+    uint256 sellFee
+  ) external onlyRiskCouncil notLocked(_gsmTimelocksByAddress[gsm].gsmFeeStrategyLastUpdated) {
+    address currentFeeStrategy = IGsm(gsm).getFeeStrategy();
+    require(currentFeeStrategy != address(0), 'GSM_FEE_STRATEGY_NOT_FOUND');
+    uint256 currentBuyFee = IGsmFeeStrategy(gsm).getBuyFee(10e5);
+    uint256 currentSellFee = IGsmFeeStrategy(gsm).getSellFee(10e5);
```

Comment:
I think i may have confused myself, i thought that this method returns the fee percentage and not the gross fee amount.
So i think the correct value is `1e4` here since 
`getBuyFee(a):`
```
GrossAmountFee = (a*buyFee)/ 1e4 
if a = 1e4
GrossAmountFee = buyFee
```